### PR TITLE
🌐 [i18n-KO] Translated guides/collections.md to Korean

### PR DESCRIPTION
**Pull request by usr-bin-ksh**
# What does this PR do?

Translated the `guides/collections.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team Hugging Face KREW, may you please review this PR? 
@harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae, @usr-bin-ksh, @fabxoe

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

File: docs/source/ko/guides/collections.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  item_id='TheBloke/U-Amethyst-20B-AWQ',
+  item_type='model',
+  position=88,
+  note=None
+)
+```
+
+[`get_collection`]에 의해 반환된 [`Collection`] 객체에는 다음이 포함되어 있습니다:
+- 높은 수준의 메타데이터: `slug`, `owner`, `title`, `description` 등
+- [`CollectionItem`] 객체의 목록; 각 항목은 모델, 데이터셋, 공간 또는 논문을 나타냅니다.
```
Comment (cjfghk5697) [inline docs/source/ko/guides/collections.md:48]:
```suggestion
- [`CollectionItem`] 객체의 목록; 각 항목은 모델, 데이터셋, Space 또는 논문을 나타냅니다.
```

원문을 보니
```
- a list of [`CollectionItem`] objects; each item represents a model, a dataset, a Space, or a paper.
```
Space라 되어있고, 위에서 Spaces라고 번역한 기록이 있어서  Space 어떨까요?

File: docs/source/ko/guides/collections.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+모든 Collection 항목에는 다음이 보장됩니다:
+- 고유한 `item_object_id`: 데이터베이스에서 Collection 항목의 id
+- 기본 항목(모델, 데이터셋, 공간, 논문)의 Hub에서의 `item_id`; 고유하지 않으며, `item_id`/`item_type` 쌍만 고유합니다.
+- `item_type`: 모델, 데이터셋, 공간, 논문
```
Comment (cjfghk5697) [inline docs/source/ko/guides/collections.md:53]:
```suggestion
- `item_type`: 모델, 데이터셋, Space, 논문
```

Space가 괜찮다면 이것도 같이 추가하면 될거 같습니다 ㅎㅎ

File: docs/source/ko/guides/collections.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+)
+```
+
+[`get_collection`]에 의해 반환된 [`Collection`] 객체에는 다음이 포함되어 있습니다:
+- 높은 수준의 메타데이터: `slug`, `owner`, `title`, `description` 등
+- [`CollectionItem`] 객체의 목록; 각 항목은 모델, 데이터셋, 공간 또는 논문을 나타냅니다.
+
+모든 Collection 항목에는 다음이 보장됩니다:
+- 고유한 `item_object_id`: 데이터베이스에서 Collection 항목의 id
+- 기본 항목(모델, 데이터셋, 공간, 논문)의 Hub에서의 `item_id`; 고유하지 않으며, `item_id`/`item_type` 쌍만 고유합니다.
```
Comment (cjfghk5697) [inline docs/source/ko/guides/collections.md:52]:
```suggestion
- 기본 항목(모델, 데이터셋, Space, 논문)의 Hub에서의 `item_id`; 고유하지 않으며, `item_id`/`item_type` 쌍만 고유합니다.
```

File: docs/source/ko/guides/collections.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+마지막으로 [`delete_collection_item`]을 사용하여 item을 제거할 수도 있습니다.
+
+```py
+>>> from huggingface_hub import get_collection, update_collection_item
+
+# Collection 가져오기
+>>> collection_slug = "osanseviero/os-week-highlights-sept-18-24-650bfed7f795a59f491afb80"
+>>> collection = get_collection(collection_slug)
+
+# 목록에서 `coqui/xtts` 공간 제거
```
Comment (cjfghk5697) [inline docs/source/ko/guides/collections.md:211]:
```suggestion
# 목록에서 `coqui/xtts` Space 제거
```

**Review by cjfghk5697 — COMMENTED** [review]
좋은 번역 감사합니다!

**Comment by heuristicwave** [comment]
[@Wauplin](https://github.com/Wauplin) I've reviewed this PR, and it looks good to me.  
Would you mind merging this PR, please?

**Review by Wauplin — APPROVED** [review]
Thanks @usr-bin-ksh and all reviewers! You're doing an amazing job! :hugs:

**Pull request merged**